.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Run Tests
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, "3.10"]
```

Comment:
Minor: I think we wrapped all of these in quotes in another repo, but not a big deal.